kaiax/gasless/impl/builder.go

```diff
@@ -30,7 +30,10 @@ func (g *GaslessModule) ExtractTxBundles(txs []*types.Transaction, prevBundles [
 	approveTxs := map[common.Address]*types.Transaction{}
 	targetTxHash := common.Hash{}
 	for _, tx := range txs {
-		addr := tx.ValidatedSender()
+		addr, err := types.Sender(g.signer, tx)
+		if err != nil {
+			panic(err)
```

Comment:
```suggestion
			return nil
```

Reply:
What about continue?
Even if one abnormal tx fails, other bundles can still be generated.

Reply:
fixed at https://github.com/kaiachain/kaia/pull/275/commits/18b2de490c0b5c019133d00da4656783223a8128